lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
We are not able to bind clobs descriptor to varchar columns. This was the
first thing I tried to keep BC.
Le mer. 29 juin 2016 à 17:19, Steve Müller <notifications@github.com> a
écrit :

> In lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php
> <https://github.com/doctrine/dbal/pull/2386#discussion_r68966187>:
>
> > @@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
> >       */
> >      public function bindValue($param, $value, $type = null)
> >      {
> > -        return $this->bindParam($param, $value, $type, null);
> > +        if ($value instanceof OCI8Descriptor) {
>
> I would try to keep it simple then and map PDO::PARAM_STR to OCI
> descriptor + OCI_B_CLOB and PDO::PARAM_LOB to OCI descriptor + OCI_BLOB
> if that works for all cases. We should able to keep complete BC and do not
> need to touch other components of DBAL in this case. What do you think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/doctrine/dbal/pull/2386/files/4e32f61517a5802ae2bea113a097d7bf168ca329#r68966187>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABQr89GYrXUVqSacaRm6MRQhz-LGay2Uks5qQoz4gaJpZM4Ib4Im>
> .
>